docs/device/install.md

```diff
@@ -37,8 +37,10 @@ brew install naisdevice
 1. [Install Kolide agent](install.md#install-kolide-agent).
 2. Add the nais PPA repo:
 ``` 
-curl -s --compressed "https://ppa.nais.io/KEY.gpg" | sudo apt-key add -
-curl -s --compressed -o "https://ppa.nais.io/nav_nais.list" | sudo tee /etc/apt/sources.list.d/nav_nais.list
+NAIS_GPG_KEY="/usr/local/share/keyrings/nav_nais.gpg"
```

Comment:
Jeg trodde konvensjonen var å legge slike keys her: `/etc/apt/trusted.gpg.d/` ?

Reply:
Fulgte denne https://unix.stackexchange.com/questions/332672/how-to-add-a-third-party-repo-and-key-in-debian/582853#582853

Har ingen sterk formening utover å hensynta dette: https://askubuntu.com/a/1307181/817936